Source/DataSources/GeoJsonDataSource.js

```diff
@@ -690,6 +690,12 @@ define([
         name : {
             get : function() {
                 return this._name;
+            },
+            set : function(value) {
```

Comment:
I need to double check in the code, but I don't think we want to raise the `_changed` event, here.  I think that event is meant for underlying Entity changes, which would basically cause more work to be done than needed if we raise it here.  But like I said, I need to check how we are using it.